apps/docs-app/docs/features/testing/vitest.md

```diff
@@ -107,6 +107,20 @@ Next, update the `test` target in the `angular.json` to use the `@analogjs/platf
 
 > You can also add a new target and name it `vitest` to run alongside your `test` target.
 
+Next, modify your `tsconfig.spec.json` to stipulate `vitest` rather than `jasmine` (or `jest`) as your test runner, and add the `test.ts` file you created above:
```

Comment:
How is this different than the text below it?

Reply:
Quite right; hadn't noticed that section below.  Will revert this addition.  However the `@nx/vite` part above still needs adding.